cmd/nfsmount/nfsmount.go

```diff
@@ -0,0 +1,50 @@
+// NFS mount is only needed for macOS since it has no
+// support for FUSE-based file systems
+//go:build darwin && !cmount
+// +build darwin !cmount
+
+package nfsmount
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"os/exec"
+	"runtime"
+
+	"github.com/rclone/rclone/cmd/mountlib"
+	"github.com/rclone/rclone/cmd/serve/nfs"
+	"github.com/rclone/rclone/vfs"
+)
+
+func init() {
+	mountlib.NewMountCommand("mount", false, mount)
+	mountlib.AddRc("mount", mount)
+}
+
+func mount(VFS *vfs.VFS, mountpoint string, opt *mountlib.Options) (asyncerrors <-chan error, unmount func() error, err error) {
+	s, err := nfs.NewServer(context.Background(), VFS, &nfs.Options{})
+	if err != nil {
+		return
+	}
+	errChan := make(chan error, 1)
+	go func() {
+		errChan <- s.Serve()
+	}()
+	_, port, err := net.SplitHostPort(s.Addr().String())
+	if err != nil {
+		err = fmt.Errorf("cannot find port number in %s", s.Addr().String())
+		return
+	}
+	err = exec.Command("mount", fmt.Sprintf("-oport=%s,mountport=%s", port, port), "localhost:", mountpoint).Run()
```

Comment:
Does this stand a chance of working on non macOS systems?

I guess in non macOS world then you could `sudo` it.

Reply:
I didn't test it. I would consider that a rarely usable feature since on other Unix platforms FUSE is working pretty well.